transformer_engine/jax/fp8.py

```diff
@@ -158,13 +169,14 @@ def initialize(margin: float = 0.0,
         FP8Helper.FWD_DTYPE, FP8Helper.BWD_DTYPE = \
             _format2dtypes(FP8Helper.FP8_FORMAT)
         FP8Helper.UPDATE_FP8META_INTERVAL = update_fp8meta_interval
-        FP8Helper.AMAX_HISTORY_SIZE = amax_history_size
+        FP8Helper.AMAX_HISTORY_LEN = amax_history_len
+        FP8Helper.AMAX_COMPUTE_ALGO = amax_compute_algo
         FP8Helper.FP8_2X_ACC_FPROP = bool(
             int(os.environ.get(FP8Helper.FP8_2X_ACC_FPROP_ENV_VAR_NAME, False)))
         FP8Helper.FP8_2X_ACC_DGRAD = bool(
-            int(os.environ.get(FP8Helper.FP8_2X_ACC_DGRAD_ENV_VAR_NAME, False)))
+            int(os.environ.get(FP8Helper.FP8_2X_ACC_DGRAD_ENV_VAR_NAME, True)))
         FP8Helper.FP8_2X_ACC_WGRAD = bool(
-            int(os.environ.get(FP8Helper.FP8_2X_ACC_WGRAD_ENV_VAR_NAME, False)))
+            int(os.environ.get(FP8Helper.FP8_2X_ACC_WGRAD_ENV_VAR_NAME, True)))
```

Comment:
Why are these knobs being exposed via envvars? I can see why we want these to be tunable in python for quick debugging without recompilation but these are supposed to be internal.

Reply:
@ksivaman ,
All envvar regarding `2xACC` are removed in commit - [e4371b9](https://github.com/NVIDIA/TransformerEngine/pull/54/commits/e4371b9048e53210265c81e1a689f1495b7028c2)